src/api/app/controllers/webui/users/notifications_controller.rb

```diff
@@ -67,6 +68,24 @@ def set_notifications
     @notifications = filter_notifications_by_kind(@notifications, @filter_kind)
   end
 
+  def set_notifications_count
+    @counted_notifications = {}
+    @counted_notifications['all'] = @notifications.count
+    @counted_notifications['unread'] = @notifications.unread.count
+    @counted_notifications['read'] = @notifications.read.count
+    @counted_notifications['comments'] = @notifications.for_comments.count
+    @counted_notifications['requests'] = @notifications.for_requests.count
+    @counted_notifications['incoming_requests'] = @notifications.for_incoming_requests(User.session).count
+    @counted_notifications['outgoing_requests'] = @notifications.for_outgoing_requests(User.session).count
+    @counted_notifications['relationships_created'] = @notifications.for_relationships_created.count
+    @counted_notifications['relationships_deleted'] = @notifications.for_relationships_deleted.count
+    @counted_notifications['build_failures'] = @notifications.for_build_failures.count
+    @counted_notifications['reports'] = @notifications.for_reports.count
+    @counted_notifications['workflow_runs'] = @notifications.for_workflow_runs.count
+    @counted_notifications['appealed_decisions'] = @notifications.for_appealed_decisions.count
```

Comment:
```suggestion
```
Shouldn't adding all the filters back have a negative inpact on the users with lots of notifications?

I would prefer to introduce all these filters gradually. I don't think this is a good idea.

Reply:
Deal https://github.com/openSUSE/open-build-service/pull/16260